Bottom: 6d6d9b2a7a94b6ead3285f418f7495b7cc33d2f1
Top:    189293318fb1a1c2266ca21babf922bca8828d65
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:43:35 +0100

Refresh of hid-gtk-rework-zoom-pan-api

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 4cd4d22..c008b25 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -122,23 +122,6 @@ ghid_zoom_view_fit (void)
                                  PCB->MaxHeight / gport->height));
 }
 
-static void
-ghid_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
-{
-  int widget_x, widget_y;
-
-  /* Work out where on the screen the flip point is */
-  ghid_pcb_to_event_coords (center_x, center_y, &widget_x, &widget_y);
-
-  ghid_flip_x = ghid_flip_x != flip_x;
-  ghid_flip_y = ghid_flip_y != flip_y;
-
-  /* Pan the board so the center location remains in the same place */
-  ghid_pan_view_abs (center_x, center_y, widget_x, widget_y);
-
-  ghid_invalidate_all ();
-}
-
 /* ------------------------------------------------------------ */
 
 static const char zoom_syntax[] =
